cluster-up/hack/common.sh

```diff
@@ -35,4 +35,4 @@ provider_prefix=${JOB_NAME:-${KUBEVIRT_PROVIDER}}${EXECUTOR_NUMBER}
 job_prefix=${JOB_NAME:-kubevirt}${EXECUTOR_NUMBER}
 
 mkdir -p $KUBEVIRTCI_CONFIG_PATH/$KUBEVIRT_PROVIDER
-KUBEVIRTCI_TAG=2104301741-c950058
+KUBEVIRTCI_TAG=2104200713-f58c002
```

Comment:
Leftovers from a rebase that went not correct, please check it

Reply:
Removed.